Bakery/DesignSystem/Components/CommonCard/CommonCardViewModelExtension.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  DSButtonViewModelExtension.swift
+//  Bakery
+//
+//  Created by Елизавета Кравченкова   on 18.05.2025.
+//
+
+import Foundation
+import UIKit
+
+
+extension CommonCardViewModel: Decodable {
+    enum CodingKeys: String, CodingKey {
+        case identifier, backroundImage, style, size, content, layout, action
+    }
+    
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        
+        identifier = try container.decode(String.self, forKey: .identifier)
+        content = try container.decode(DSContainerViewModel.self, forKey: .content)
+        style = try container.decode(CommonCardStyle.self, forKey: .style)
+        size = try container.decode(CommonCardSize.self, forKey: .size)
+        
+        let imageBase64 = try container.decode(String.self, forKey: .backroundImage)
+        if let data = Data(base64Encoded: imageBase64) {
+            let ui_image = UIImage(data: data)
```

Comment:
```suggestion
            let uiImage = UIImage(data: data)
```

ну или как-то иначе) snake_case в iOS крайне не принят)